cmake/cpu_extension.cmake

```diff
@@ -84,7 +84,11 @@ endif()
 
 message(STATUS "CPU extension compile flags: ${CXX_COMPILE_FLAGS}")
 
-list(APPEND LIBS dnnl numa)
+list(APPEND LIBS numa)
+
+if (NOT POWER9_FOUND AND NOT POWER10_FOUND)
```

Comment:
wondering if we can add some comment here why are we doing this when this condition will be removed?

Reply:
Why don't we just make a check for x86 for this condition?

Reply:
Added a comment and updated check based on `AVX2` or `AVX512_FOUND`.